hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/ozone/scm/node/TestSCMNodeMetrics.java

```diff
@@ -182,7 +196,7 @@ public void testNodeCountAndInfoMetricsReported() throws Exception {
 
     MetricsRecordBuilder metricsSource = getMetrics(SCMNodeMetrics.SOURCE_NAME);
 
-    assertGauge("HealthyNodes", 1, metricsSource);
```

Comment:
Can we change the test so that it still verifies the presence of "HealthyNodes" in the SCM eventually?

Reply:
yup. done.